app/web/src/components/IdeSideBar/sidebarConfig.tsx

```diff
@@ -0,0 +1,156 @@
+import React, { ReactNode } from 'react'
+import { SvgNames } from 'src/components/Svg/Svg'
+import { useIdeContext } from 'src/helpers/hooks/useIdeContext'
+
+interface SidebarConfigType {
+  name: string
+  icon: SvgNames
+  disabled: boolean
+  panel: ReactNode | null
+}
+
+export const sidebarTopConfig: SidebarConfigType[] = [
+  {
+    name: 'Files',
+    icon: 'files',
+    disabled: false,
+    panel: (
+      <article className="px-2 py-4">
+        <p>
+          <em>Coming Soon</em>
+        </p>
+        <hr className="my-4" />
+        <p>
+          We're working on multi-file support in tandem with the GitHub
+          integration.
+        </p>
+      </article>
+    ),
+  },
+  {
+    name: 'GitHub',
+    icon: 'github',
+    disabled: false,
+    panel: (
+      <article className="px-2 py-4">
+        <p>
+          <em>Coming Soon</em>
+        </p>
+        <hr className="my-4" />
+        <p>
+          This integration will allow you to sync a project with a GitHub repo
+          and push changes back to it as a commit!
+        </p>
+      </article>
+    ),
+  },
+  {
+    name: 'Visibility',
+    icon: 'eye',
+    disabled: true,
+    panel: null,
+  },
+]
+
+const DiscordLink = () => (
+    <a className="underline text-ch-pink-300"
+        href="https://discord.gg/SD7zFRNjGH"
+        target="_blank"
+        rel="noreferrer"
+    >
+        Discord
+    </a>
```

Comment:
Feels silly to put this here for temporary copy, but I really didn't want to rewrite the Discord link 3 times lol. Should I have done this differently?